config/default.js

```diff
@@ -220,7 +220,12 @@ module.exports = {
         process.env.LIQUIDITY_PROVIDER_ADDRESS || '0x4789FD18D5d71982045d85d5218493fD69F55AC4',
     },
     pkds: {
-      user1: '0x1ac3b61ecba1448e697b23d37efe290fb86554b2f905aaca3a6df59805eca366',
+      user1:
```

Comment:
There are keys already under TEST_OPTIONS: { signingKeys : .... }} in config. Including mnemomnic and ethereum addresses